benches/aarch64/mod.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+extern crate criterion;
+extern crate linux_loader;
+extern crate vm_memory;
+
+use linux_loader::configurator::BootParams;
+use vm_memory::{ByteValued, GuestAddress, GuestMemoryMmap};
+
+const MEM_SIZE: usize = 0x100_0000;
+const FDT_MAX_SIZE: usize = 0x20;
+
+pub fn create_guest_memory() -> GuestMemoryMmap {
```

Comment:
I think we could add an arch-independent `utils` module, it just seemed overkill to do so for a one-liner function (though I think it will become necessary with the next arch-independent utility). Would you prefer that I add it now or leave it for later?

Reply:
Would be tidier to add it right now, but it's by no means a blocker :D